Formula/davix.rb

```diff
@@ -27,7 +27,7 @@ class Davix < Formula
   end
 
   def install
-    system "cmake", ".", *std_cmake_args
+    system "cmake", ".", *std_cmake_args, "-DCMAKE_EXE_LINKER_FLAGS=-Wl,-rpath,#{rpath}"
```

Comment:
```suggestion
    system "cmake", ".", *std_cmake_args, "-DCMAKE_INSTALL_RPATH=#{rpath}"
```
Does this not work?

Reply:
This I got from my [codec2 PR](https://github.com/Homebrew/homebrew-core/pull/86210/commits/62f782b9c3eb642d1d88f3a814a81c36f1b53b8b)

Reply:
We should try to use `CMAKE_INSTALL_RPATH` if we can -- more formulae use it. It also works for both libraries and executables, whereas `CMAKE_EXE_LINKER_FLAGS` only applies to executables.